server/text_generation_server/models/__init__.py

```diff
@@ -110,6 +111,138 @@
     __all__.append(Mamba)
 
 
+class ModelType(enum.Enum):
+    MAMBA = {
+        "type": "ssm",
```

Comment:
To be consistent with `transformers` https://huggingface.co/state-spaces/mamba-130m-hf/blob/main/config.json

```suggestion
        "type": "mamba",
```

Reply:
Indeed.

Reply:
This is the original one: https://huggingface.co/state-spaces/mamba-2.8b-slimpj